packages/design-system/src/components/Link/Link.tsx

```diff
@@ -92,7 +92,7 @@ const Link = React.forwardRef(
 						<Icon
 							className="link__icon link__icon--before"
 							name={icon}
-							data-testid="link.icon.before"
+							data-test="link.icon.before"
```

Comment:
why this change? what is the difference between data-test and data-testid otherwise?

Reply:
https://github.com/Talend/ui/blob/master/packages/design-system/adr/data-test.md

`data-test` is for internal use and is the basic data-attribute for testing in the DS. `data-testid` is reserved to product teams (ie: `<Link href="/pouet" data-testid="link.ctaForPouet">Let's pouet</Link>`).